app/src/main/java/com/duckduckgo/app/global/db/AppDatabase.kt

```diff
@@ -693,6 +688,15 @@ class MigrationsProvider(val context: Context, val settingsDataStore: SettingsDa
         }
     }
 
+    private val MIGRATION_57_TO_58: Migration = object : Migration(57, 58) {
+        override fun migrate(database: SupportSQLiteDatabase) {
+            database.execSQL(
+                "DROP TABLE IF NOT EXISTS `refreshes` (`id` INTEGER PRIMARY KEY AUTOINCREMENT NOT NULL, " +
```

Comment:
DROP TABLE IF EXISTS, otherwise, there's nothing to drop

Reply:
🤦‍♀️